ElectronicObserver/Data/EquipmentDataMaster.cs

```diff
@@ -341,7 +341,9 @@ public bool IsAntiSubmarineAircraft
 	/// <summary> 水上電探かどうか </summary>
 	public bool IsSurfaceRadar => IsRadar && LOS >= 5;
 
+	public bool IsHighAccuracyRadar => IsRadar && Accuracy >= 8;
```

Comment:
Might wanna use
```cs
this.IsHighAccuracyRadar()
```
here to avoid logic duplication.

Reply:
done